ROS/Chapter_6.ipynb

```diff
@@ -0,0 +1,1208 @@
+{
```

Comment:
<p>We could use the same order for the import as the one we use in pymc3-resources</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bambinos/Bambi_resources/pull/7/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
Fixed

Reply:
Im pretty sure